pkgs/ffigen/lib/ffigen.dart

```diff
@@ -7,6 +7,5 @@
 /// See complete usage at - https://pub.dev/packages/ffigen.
```

Comment:
This doc comment should be updated that this is to use FfiGen from Dart. And Point to the FfiGen class.

And then some reference to the README for yaml and `dart run` based usage.

If we want to go FfiGen from Dart first (e.g. users should write `tool/ffigen.dart` that has the config in Dart) we should probably rewrite the readme as well to not have the YAML config so front-and-center. Can be done in a follow up PR.